torchbench.py

```diff
@@ -839,7 +920,7 @@ def run_one_model(
             # check correctness.
             # TODO(jansel): submit upstream fix for this
             pass
-        elif not same(correct_result, new_result):
+        elif not same(correct_result, new_result, cos_similarity):
```

Comment:
If we let the fp32 vision models to run, their output is fp32 and reference model output is fp16. They can not be compared.

Reply:
I'm suggesting we still exclude them.  Just only exclude them when running fx2trt.

Reply:
yea, I changed the previous comment that should exclude them.